### PR TITLE
Option to disable autocompletion [API-1457]

### PR DESCRIPTION
**Pull request by utku-caglayan**
Provide a flag and a configuration item to disable the autocompletion on interactive mode

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -49,16 +49,18 @@ type SSLConfig struct {
 }
 
 type Config struct {
-	Hazelcast hazelcast.Config
-	SSL       SSLConfig
+	Hazelcast             hazelcast.Config
+	SSL                   SSLConfig
+	DisableAutocompletion bool
```
Comment (yuce) [inline config/config.go:54]: How about `NoAutocompletion`?

Comment (yuce) [inline config/config.go:54]: How about `NoAutocompletion`?

Comment (utku-caglayan) [reply config/config.go]: Sure, also renamed the flag to `no-completion` https://github.com/hazelcast/hazelcast-commandline-client/pull/102/commits/b357d82b69f70aa42dcea4c6664b450ada4262f1

File: config/config.go [hunk 2]
```diff
@@ -49,16 +49,18 @@ type SSLConfig struct {
...
-	Cluster string
-	Token   string
-	Address string
-	Verbose bool
+	CfgFile               string
+	Cluster               string
+	Token                 string
+	Address               string
+	Verbose               bool
+	DisableAutocompletion bool
```
Comment (yuce) [inline config/config.go:63]: Would it make sense to change this one to match `NoAutocompletion`?

**Comment by yuce** [comment]
@utku-caglayan Could you also update the documentation?

**Review by yuce — APPROVED** [review]
Looks good!

**Pull request merged**